app/src/main/java/com/ayal/todo/DataManager.java

```diff
@@ -0,0 +1,15 @@
+package com.ayal.todo;
+
+import java.util.ArrayList;
+
+public class DataManager {
+    private static ArrayList<Todo> todos = new ArrayList<>();
+
+    public static void AddToList(Todo task){
```

Comment:
method names always should start with lower case

Reply:
opps